.rubocop.yml

```diff
@@ -1,7 +1,18 @@
 inherit_from: .rubocop_todo.yml
+Style/StringLiteral:
+  Enabled: true
+  EnforcedStyle: single_quotes
```

Comment:
Turns out Hound and Rubocop have different defaults. So instead of disabling the check entirely, I'm explicitly forcing it to the Rubocop default, which should make Hound use that style as well.

Weirdly, Hound hasn't finishing it's scan. Something to do with me amending and force-pushing, perhaps?